item.rb

```diff
@@ -0,0 +1,25 @@
+class Item
+  attr_reader :archived, :label, :author, :genre
+  attr_accessor :publish_date
+
+  def initialize(publish_date)
+    @id = Random.rand(1...1000)
+    @archived = false
+    @publish_date = publish_date
+  end
+
+  def can_be_archived?
+    diff = Time.now.year - @publish_date
+
+    diff > 10
+  end
```

Comment:
[OPTIONAL] - Not critical

## Hello @levy002 :wave: 
This is very good. I almost had a different approach but on checking the [project's](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md) requirements I realized mine could have been awfully wrong. 
Either way, here is what I had in mind

```  
def initialize(publish_date)
    @id = Random.rand(1...1000)
    @archived = true
    @publish_date = publish_date
  end

  def can_be_archived?
    diff = Time.now.year - @publish_date

    @archived = false unless diff > 10
  end
```

Again, good job :+1: